source/tutorial/install-mongodb-on-windows.txt

```diff
@@ -44,10 +44,17 @@ enter the following command in either the :guilabel:`Command Prompt` or
    OS Version:                10.0.17134 N/A Build 17134
    System Type:               x64-based PC
 
-.. important:: If you are running any edition of Windows Server 2008
-   R2 or Windows 7, please install `a hotfix to resolve an issue with
-   memory mapped files on Windows 
-   <http://support.microsoft.com/kb/2731284>`_.
+.. important:: 
+
+   Before installing MongoDB, update your Windows host with the latest system updates. Updates related to MongoDB include:
```

Comment:
The last bit here flows weird into the bullets. I'd end with 'update...with the latest system updates. Ensure the following updates are installed on the host:"